pkg/datasource/csvsource/exchange_tick_downloader_test.go

```diff
@@ -0,0 +1,28 @@
+package csvsource
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
I would recommend disabling the test not to put strain on the resources unecessarily